cmd/govim/oswatcher.go

```diff
@@ -0,0 +1,47 @@
+// +build !darwin
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/fsnotify/fsnotify"
+)
+
+func newOSWatcher(gomodpath string) (*osWatcher, error) {
+	mw, err := fsnotify.NewWatcher()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create new watcher: %v", err)
+	}
+
+	eventCh := make(chan fsEvent)
+	go func() {
```

Comment:
We should pass a `*gopkg.in/tomb.v2.tomb.Tomb` to `NewOSWatcher` (assuming we put this in an internal package) and then spawn this go routine using `*gopkg.in/tomb.v2.tomb.Tomb.Go()`. 

The approach is not perfect, but it's helped track down errors in the past

Reply:
Ok! - I'll add tomb